package-lock.json

```diff
@@ -21614,9 +23667,10 @@
       }
     },
     "node_modules/rollup-plugin-terser/node_modules/terser": {
-      "version": "5.19.2",
+      "version": "5.27.0",
```

Comment:
Looks like this terser update might be responsible for the bundle size change. I tried updating typescript alone and it didn't change the output but this branch does have a diff. Diff looks innocuous. Somehow previous version lucked out and didn't require an alias for `H`, but after update it does.
<img width="1499" alt="image" src="https://github.com/lit/lit/assets/40413829/77e40cde-9656-4346-ae08-98f7042a7ccd">